### PR TITLE
support sync ollama embeddings

### LINKED ISSUE #3387: [Bug]: Embedding Function on Ollama Seems not Working
### What happened?

Embedding functionality seems not working for Ollama. I have `nomic-embed-text` installed and tested using Ollama Python library..

`response = embedding(model='ollama/nomic-embed-text',
                     input=["good morning from litellm"],
                     api_base="http://localhost:11434")
print(response)`

I don't know if I am using embedding function incorrectly or there is another issue (lack of Ollama support for embeddings, bug etc.)

### Relevant log output

```shell
File "/home/****/codes/python/experiment-litellm/main.py", line 41, in <module>
    response = embedding(model='ollama/nomic-embed-text',
  File "/home/****/miniconda3/envs/experiment-litellm/lib/python3.8/site-packages/litellm/utils.py", line 3129, in wrapper
    raise e
  File "/home/****/miniconda3/envs/experiment-litellm/lib/python3.8/site-packages/litellm/utils.py", line 3072, in wrapper
    result._response_ms = (
AttributeError: 'NoneType' object has no attribute '_response_ms'
```

### Twitter / LinkedIn details

_No response_

### PR DESCRIPTION
**Pull request by mbektas**
## Support for ollama sync embeddings

Fixes #3387 and #2712

🐛 Bug Fix

## OS Tests (optional but appreciated):

- [ ] Tested on Windows
- [x] Tested on MacOS
- [ ] Tested on Linux

### CODE REVIEW

File: litellm/tests/test_ollama_local.py [hunk 1]
```diff
@@ -24,6 +24,14 @@
 
 # asyncio.run(test_ollama_aembeddings())
 
+# def test_ollama_embeddings():
```
Comment (ishaan-jaff) [inline litellm/tests/test_ollama_local.py:27]: Can we add a test for this - I believe you can create a mock test for a stubbed ollama endpoint 

want to make sure there are no regressions on this

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
can you please add testing for this scenario @mbektas

File: litellm/tests/test_ollama_local.py (hunk 1 above)
Comment (mbektas) [reply litellm/tests/test_ollama_local.py:27]: added unit tests to test_ollama.py as well. however, I am not clear why test method calls are commented out.

Comment (ishaan-jaff) [reply litellm/tests/test_ollama_local.py:27]: > however, I am not clear why test method calls are commented out.

Those test call ollama locally and are not mock requests

Comment (mbektas) [reply litellm/tests/test_ollama_local.py:27]: tests in `litellm/tests/test_ollama.py` don't hit ollama server. so we can uncomment all of those.

**Review by ishaan-jaff — APPROVED** [review]

**Pull request merged**